github/libp2p.yml

```diff
@@ -5592,6 +5592,16 @@ teams:
       member:
         - mxinden
     privacy: closed
+  universal-connectivity Maintainers:
+    create_default_maintainer: false
+    description: Maintainers of universal-connectivity that are not maintainers of
+      other libp2p implementations
+    members:
+      maintainer:
+        - dhuseby
+      member:
+        - kentbull
```

Comment:
FYI, only organization members can be team members

Reply:
added kentbull to the org members list.